### ISSUE TITLE
Missing accessibility / Disclaimer / Privacy & cookies links in footer

### ISSUE THREAD

**Issue by mjpritchard** [cedadev/ceda-div-hugo-hinode#31]

**Comment by gap736uk** [comment]
Copies from Slack 

The standard footer had these links in :

[Accessibility](http://www.ceda.ac.uk/accessibility) | [Disclaimer](http://www.ceda.ac.uk/disclaimer) | [Privacy and Cookies](http://www.ceda.ac.uk/privacy-and-cookies/)

These aren't on the new div site at present.... and the links above are presently broken (they were taken from the standard footer used on the Archive sites (e.g. data.ceda, catalogue. ceda etc) .

I think originally they linked through to articles listed on this part of the help.ceda pages:
https://help.ceda.ac.uk/category/4858-policy-documents

The location on help.ceda.ac.uk is the place to link to and for us to maintain a common set of policy docs.

**Comment by danny-lloyd** [comment]
Fixed by https://github.com/cedadev/ceda-div-hugo-hinode/commit/2704bb163181fd7600b24b7035a90f29f0f07ed7
